packages/stencil-components/package.json

```
@@ -0,0 +1,26 @@
+{
+  "name": "@okta/stencil.components",
```

```suggestion
  "name": "@okta/odyssey.web-components",
```

I would also like to see the package name changed to `odyssey-web-components` or something similar since that is what is consumed by the end user.

---

Stencil here refers to the **type** of component we're building. In this case, [Stencil](https://stenciljs.com/docs/introduction) is the tool for how we're compiling our Web components. Therefore, I'd assume we want to avoid name-spacing these with `odyssey` unless we're **only going to ship one component library**.

I foresee a similar convention elsewhere:
- `@okta/react.components`
- `@okta/vue.components`
- `@okta/polymer.components`

---

Fair point. The convention you makes sense, still I have a strong bias towarrds keeping `odyssey` in the package name, maybe prefixing the package name with it?  At the end of the day, they are frameworks providing Odyssey components.

---

Agreed @arnoldsandoval-okta - even Carbon's `cli-reporter` package is bundled under their namespace to keep ownership and scope clear.

---

Ownership currently resides within the scoped `@okta` namespace, not `@odyssey`. I'm not a huge fan of `@okta/odyssey-${somePackage}-components`, as it's far too long of a name.

@edburyenegren-okta and @arnoldsandoval-okta - We could also leverage [scoped package](https://docs.npmjs.com/using-npm/scope.html) (`@odyssey/react`) to make this even more clear. Thoughts?

---

This makes sense, feels like the right way to go imo!